examples/swift/hello_world/AsyncDemoFilter.swift

```diff
@@ -0,0 +1,52 @@
+import Envoy
+import Foundation
+
+class AsyncDemoFilter: AsyncResponseFilter {
+  private var internalCallbacks: ResponseFilterCallbacks?
+  private lazy var callbacks: ResponseFilterCallbacks = {
+    internalCallbacks! //swiftlint:disable:this force_unwrapping
+  }()
```

Comment:
```suggestion
  private var callbacks: ResponseFilterCallbacks!
```

Reply:
Nice, I figured you must know a more idiomatic way of doing this. :)